fedimint-server/src/db.rs

```diff
@@ -339,6 +366,19 @@ mod fedimint_migration_tests {
                             DbKeyPrefix::ConsensusUpgrade => {
                                 assert!(dbtx.get_value(&ConsensusUpgradeKey).await.is_some());
                             }
+                            DbKeyPrefix::ClientConfigDownload => {
+                                let downloads = dbtx
+                                    .find_by_prefix(&ClientConfigDownloadKeyPrefix)
+                                    .await
+                                    .collect::<Vec<_>>()
+                                    .await;
+                                let _downloads_len = downloads.len();
+                                // FIXME: Why does this fail?
```

Comment:
@m1sterc001guy I could use your help figuring out why this doesn't pass.

Reply:
You just need to re-create the backup databases. You can do this by deleting the "global" folder underneath db/, modify the `create_db_with_v0_data` function to add a dummy version of your new db entry, then re-run `cargo test`.

Alternatively you can write a db migration to backup your new db entry but that's probably overkill right now.